### PR TITLE
[DEV-5] Implement Cycle View Page

### PR DESCRIPTION
**Pull request by jpoist97**
Implemented Cycle view page. Added Montseratt fonts. Added styled-components as an alternative to stylesheets. Want it for color based on props here and for styling based on props in log workout component.

IMG_3727

### CODE REVIEW

File: components/cycles/Cycles.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { SafeAreaView, TouchableOpacity, Text } from 'react-native';
+import styled from 'styled-components/native';
+import { Entypo } from '@expo/vector-icons';
+import AllCycles from './AllCycles';
+
+const PlusButton = styled(TouchableOpacity)`
```
Comment (ctperry0301) [inline components/cycles/Cycles.jsx]: Maybe PlusButton Could be a component?

File: components/cycles/AllCycles.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      onIconPress={item.onIconPress}
+      onPress={item.onPress}
+      color={item.color}
+    />
+  );
+
+  return (
+    <View>
+      <Title>Cycles</Title>
+      <FlatList
```
Comment (ctperry0301) [inline components/cycles/AllCycles.jsx:30]: Does FlatList allow you to scroll in the case a few more cycles are added?

File: App.js [hunk 3]
```diff
@@ -9,8 +9,31 @@ import Calendar from './components/calendar/Calendar';
 import Home from './components/home/Home';
 import Profile from './components/profile/Profile';
 import Workouts from './components/workouts/Workouts';
+import Cycles from './components/cycles/Cycles';
 import { configureFonts, DefaultTheme, Provider as PaperProvider } from 'react-native-paper';
-
+import { Entypo } from '@expo/vector-icons';
+import { AppLoading } from 'expo';
+import {
+  useFonts,
```
Comment (ctperry0301) [inline App.js:17]: Not a Huge Deal, but we should get rid of fonts we wont use as each font slightly increase the time the app takes to load. (Like we probably arent ever going to use the italic ones or the super thin and thick ones)

**Review by ctperry0301 — DISMISSED** [review]
Looks really good!

File: components/cycles/Cycles.jsx (hunk 1 above)
Comment (jpoist97) [reply components/cycles/Cycles.jsx]: Yeah good idea, I'll go and add that

File: components/cycles/AllCycles.jsx (hunk 2 above)
Comment (jpoist97) [reply components/cycles/AllCycles.jsx:30]: Yep, it'll scroll

File: App.js (hunk 3 above)
Comment (jpoist97) [reply App.js:17]: Yeah good point, should we just leave them all for now and throw a task up for once we have design finalized to remove extras?

Comment (finlaylp) [reply App.js:17]: Yeah agreed, let's make that a new task and edit out the ones we won't use later on, it isn't very urgent.

File: components/cycles/Cycles.jsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+   padding: 10px;
+`;
+
+const Buttontext = styled.Text`
+   font-family: 'Montserrat_500Medium';
+   font-size: 18px;
+`;
+
+export default () => {
+  const items = [
```
Comment (finlaylp) [inline components/cycles/Cycles.jsx:14]: Is this temporary or are these gonna be default placeholder cycle cells?

**Review by finlaylp — DISMISSED** [review]
Looks great.

File: components/cycles/Cycles.jsx (hunk 4 above)
Comment (jpoist97) [reply components/cycles/Cycles.jsx:14]: These are temporary, they'll be replaced by a call to the database. I don't think we should have placehodler cells if a user does not have any cycles

Comment (finlaylp) [reply components/cycles/Cycles.jsx:14]: Great was thinking the same thing

**Review by ctperry0301 — APPROVED** [review]

**Pull request merged**